tools/bgfx.natvis

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+    <Type Name="bx::StringView">
+        <DisplayString>{m_ptr,[m_len]s8}</DisplayString>
+        <StringView>m_ptr,[m_len]s8</StringView>
```

Comment:
If StringView s are supposed to be ASCII instead of UTF8, you might want to drop the s8 here.

Reply:
.natvis for bx types would be better.

Reply:
Separated out into https://github.com/bkaradzic/bx/pull/179 , leaving as UTF8.